app/Entities/Place/Place.php

```diff
@@ -68,4 +68,17 @@ public function features()
     {
         return $this->belongsToMany(PlaceFeature::class, 'places_places_features');
     }
+
+    public static function boot()
```

Comment:
Oh no ))
We will loose too much data, please remove. We will think about cleaning the data in deferent place, nice idea also.